File: app/src/java/java/com/example/ama/viewer/di/modules/FragmentModule.java
package com.example.ama.viewer.di.modules;

import com.example.ama.viewer.presentation.profile.ProfileFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public interface FragmentModule {

    @ContributesAndroidInjector
    ProfileFragment bindProfileFragment();
}

File: app/src/kotlin/java/com/example/ama/viewer/di/modules/FragmentModule.kt
package com.example.ama.viewer.di.modules

import com.example.ama.viewer.presentation.profile.ProfileFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
interface FragmentModule {

    @ContributesAndroidInjector
    fun bindProfileFragment(): ProfileFragment
}